### ISSUE TITLE
HDB++ clients

### ISSUE THREAD

**Issue by piogor** [3Controls/tangobox#18]
Follows #17 
The clients 
- [ ] Configuration tool installed on VM
- [ ] View/browser application installe on VM
- [ ] Example archiving configured
- [ ] History logged (checked with browser application)

**Comment by stanislaw47** [comment]
Regarding HdbConfigurationGUI: 


I have follwoed instructions presented [here](http://www.esrf.eu/computing/cs/tango/tango_doc/tools_doc/hdb++-configurator/introduction.html#tools-introduction) but when I try to execute last command, I just get:
```
Error: Could not find or load main class for org.tango.hdb_configurator.configurator.HdbConfigurator
```
@andygotz do you have any idea what has possibly gone wrong?

**Comment by bourtemb** [comment]
I advise you to use a script to start up the HDBConfiguration GUI.
Did you specify the absolute path of every Jar file in your classpath?
Do you have all the jar files described on the documentation?
If you have more problems, could you please send the script you are using.
Try to print your CLASSPATH too before launching the GUI with java. Send the CLASSPATH here if you have a problem please...

**Comment by stanislaw47** [comment]
Hi @bourtemb 
I've exported CLASSPATH with absolute paths to all needed jar files. To give more detail, I use hdb-configurator-2.2.jar from official Tango website. Maybe this version is too old?

**Comment by bourtemb** [comment]
There is a more recent version on Bintray (3.0):
https://bintray.com/tango-controls/maven/hdbpp-configurator

**Comment by stanislaw47** [comment]
Thank you @bourtemb, I've managed to succesfully launch HDBConfigurator but there are two, probably linked, issues:
- I've downloaded version 3.0 from bintray but after launch, it says it's version 2.2
- when I try to add attribute to subscription, sometimes it says `Event properties wrongly configured` or `No HDB++ library`. What is interesting, attributes of boolean type are subscribed properly. 

I'll appreciate any help.
